dts/bindings/pinctrl/telink,b91-pinctrl.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2022, Telink Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Telink B91 Pin controller node.
+    Based on pincfg-node.yaml binding.
```

Comment:
Please extend this description to detail pinctrl usage on this platform. See for example https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/pinctrl/gd%2Cgd32-pinctrl-af.yaml or https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/pinctrl/renesas%2Crcar-pfc.yaml